src/auth/views/MagicLink.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+
+import { Alert, Link, Stack, TextField, Typography } from "@mui/material";
+import { useGetLink } from "auth/hooks/mutations/useGetLink";
+import { LoadingButton } from "@mui/lab";
+import { AuthContainer } from "auth/views/components/AuthContainer";
+import { useHistory } from "react-router-dom";
+
+export function MagicLink() {
+  const [email, setEmail] = useState("");
+  const [requested, setRequested] = useState(false);
+  const history = useHistory();
+
+  const { link, loading, error } = useGetLink({
```

Comment:
nit: maybe `requestLink` would be a better name than `link`?  `link` implies to me that's a string containing the link, not a function that will request the link.